site/src/ThemeSelector.css

```diff
@@ -0,0 +1,36 @@
+.theme-selector {
+  max-width: 400px;
+  background-color: rgba(255, 255, 255, 0.9);
+  border-radius:5px;
+  color: black;
+  font-size:0;
+  outline: none;
+  display:flex;
+  max-height: 32px;
+  margin-top:3px;
+}
+
+.theme-dark .inspector-panel {
+  background-color: rgba(0, 0, 0, 0.7);
+  color: white;
+}
+
+div.layer-control {
+  width:32px;
+  height:32px;
+  justify-content: center;
+  display: flex;
+}
+img.layer-control {
+  background: white; 
+  width:24px;
+  justify-content: center;
+}
+
+svg.icon-layers > path.primary {
```

Comment:
Here's those css specifiers that I was telling you about. I specifically look for an svg of class icon-layers, then target the 'primary' path within to change its color.